lldb/source/Plugins/Language/Swift/SwiftRuntimeFailureRecognizer.h

```diff
@@ -0,0 +1,49 @@
+//===-- SwiftRuntimeFailureRecognizer.cpp -----------------------*- C++ -*-===//
```

Comment:
the `-*- C++ -*-` marker only makes sense for .h file where it is ambiguous whether it's a C or a C++ header.

Reply:
Oh... it's the filename that's wrong here!